dlc/src/main/scala/org/bitcoins/dlc/BinaryOutcomeDLCClient.scala

```diff
@@ -518,10 +518,11 @@ case class BinaryOutcomeDLCClient(
     val fundingInput =
       TransactionInput(fundingOutPoint, EmptyScriptSignature, sequence)
 
-    val unsignedTx = BaseTransaction(TransactionConstants.validLockVersion,
-                                     Vector(fundingInput),
-                                     outputs,
-                                     timeouts.contractMaturity.toUInt32)
+    val unsignedTx = BaseTransaction(
+      TransactionConstants.validLockVersion,
+      Vector(fundingInput),
+      outputs.filter(_.value >= Policy.dustThreshold),
```

Comment:
So i'm torn on this. I don't really like removing outputs silently from a transaction, rather I would rather have the caller of this API have to explicitly remove the output themselves. When we start removing outputs internally it makes it easy to fudge something up and pay a large fee.

Reply:
At some point me and Tibo plan on explicitly putting the algorithms for doing this as well as fee estimation up on the spec, as all parties must be doing things deterministically (and so allowing users to specify is off the table). But as per the discussion above I would view this in its current form as more of a place-holder (whose main function is to wipe 0 sat outputs so that tests pass as that is a common case)